docs/WebEditorBasics/WebPlayMode.md

```diff
@@ -0,0 +1,53 @@
+---
+id: WebPlayMode
+title: Web Play Mode
+---
+
+Don't always have access to a device? Waiting for a battery to recharge? No problem!
+
+You can continue to build with Enklu Cloud's blazing fast iteration times by using Web Play Mode to test your experiences.
```

Comment:
I feel like you probably don't need line 6-8, since it's kinda reiterated below under "what is web play mode"